### ISSUE TITLE
SSH authentication doesn't work

### ISSUE THREAD

**Issue by farkasmate** [rancher/terraform-controller#58]
The controller doesn't work with key-authenticated private repositories.

## Example

```
apiVersion: terraformcontroller.cattle.io/v1
kind: Module
metadata:
  name: test-module
  namespace: terraform-controller
spec:
  git:
    url: git@github.com:farkasmate/tf-test.git
    secretName: test-key

---
apiVersion: v1
kind: Secret
metadata:
  name: test-key
  namespace: terraform-controller
type: Opaque
data:
  ssh-privatekey: <snip>
```

## Behaviour

- The controller parses the k8s secret holding the SSH private key, but fails to persist it due to passing value in `fromSecret` rather than a pointer
- There's a typo in the environment variable name: `GIT_SSH_COMAND`

**Comment by k0da** [comment]
This should be fixed now

**Comment by farkasmate** [comment]
Fixed by #70
